system/doc/reference_manual/expressions.xml

```diff
@@ -2076,58 +2093,69 @@ end</pre>
+        <cell align="left" valign="middle">Left-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">+ - bor bxor bsl bsr or xor</cell>
-        <cell align="left" valign="middle">Left associative</cell>
+        <cell align="left" valign="middle">Left-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">++ --</cell>
-        <cell align="left" valign="middle">Right associative</cell>
+        <cell align="left" valign="middle">Right-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">== /= =&lt; &lt; >= > =:= =/=</cell>
-        <cell align="left" valign="middle">&nbsp;</cell>
+        <cell align="left" valign="middle">Non-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">andalso</cell>
-        <cell align="left" valign="middle">&nbsp;</cell>
+        <cell align="left" valign="middle">Left-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">orelse</cell>
+        <cell align="left" valign="middle">Left-associative</cell>
+      </row>
+      <row>
+        <cell align="left" valign="middle">catch</cell>
         <cell align="left" valign="middle">&nbsp;</cell>
       </row>
       <row>
         <cell align="left" valign="middle">= !</cell>
-        <cell align="left" valign="middle">Right associative</cell>
+        <cell align="left" valign="middle">Right-associative</cell>
       </row>
       <row>
         <cell align="left" valign="middle">?=</cell>
-        <cell align="left" valign="middle">&nbsp;</cell>
-      </row>
-      <row>
-        <cell align="left" valign="middle">catch</cell>
-        <cell align="left" valign="middle">&nbsp;</cell>
+        <cell align="left" valign="middle">Non-associative</cell>
       </row>
       <tcaption>Operator Precedence</tcaption>
     </table>
+    <change><p>Before Erlang/OTP 24, the <c>catch</c> operator had the lowest
+    precedence.</p></change>
     <note><p>The <c>=</c> operator in the table is the
     <seeguide marker="#match_operator">match operator</seeguide>.
     The character <c>=</c> can also denote the
     <seeguide marker="#compound_pattern_operator">compound pattern operator</seeguide>,
-    which can only be used in patterns.</p></note>
+    which can only be used in patterns.</p>
+    <p><c>?=</c> is restricted in that it can only be used at
+    the top-level inside a <c>maybe</c> block.</p>
+    </note>
     <p>When evaluating an expression, the operator with the highest
-      priority is evaluated first. Operators with the same priority
-      are evaluated according to their associativity.</p>
-      <p><em>Example:</em></p>
-      <p>The left associative arithmetic operators are evaluated left to
+      precedence is evaluated first. Operators with the same precedence
+      are evaluated according to their associativity. Non-associative
+      operators cannot be combined.</p>
```

Comment:
It would be nice to have a clarification of "combined" such as
...` be combined (with operator of the same precedence).`

Reply:
OK. I've added that clarification.